workflows/diagnostics/fv3net/diagnostics/prognostic_run/metrics.py

```diff
@@ -150,8 +149,12 @@ def time_and_global_mean_value(diags, mask_type=mask_type):
         if len(time_mean_value) == 0:
             return xr.Dataset()
         masked_area = _mask_array(mask_type, diags["area"], diags["land_sea_mask"])
+        if "ncol" in diags["area"].dims:
+            horizontal_dims = ["ncol"]
+        else:
+            horizontal_dims = ["x", "y", "tile"]
         time_and_global_mean_value = weighted_mean(
-            time_mean_value, masked_area, HORIZONTAL_DIMS
+            time_mean_value, masked_area, horizontal_dims
```

Comment:
I think we should really try to avoid these deep `if` statements. Since this PR is just to make the offline diagnostics work for SCREAM, I suggest just leaving this hard-coded to `HORIZONTAL_DIMS_FV3`. Same for other cases in this module.